### DISCUSSION TITLE
Removing comments from individual pages like About

### DISCUSSION THREAD

**Discussion by Mecanik** [razonyang/hugo-theme-bootstrap#545] [Q&A] [answered]
Hi,

As the title suggests, how can one remove the comments from individual pages?

I have tried bluntly comments = false but no effect.

Thanks!

**Comment by razonyang** [comment, marked as answer]
It's a typo, the parameter should be `comment`.

**Reply by Mecanik** [reply]
Great, thanks! (you might want to rename it to comments).
